data_structures/src/error.rs

```diff
@@ -262,10 +262,14 @@ pub enum TransactionError {
     ValueTransferWeightLimitExceeded { weight: u32, max_weight: u32 },
     /// Data Request weight limit exceeded
     #[fail(
-        display = "Data Request Transaction weight ({}) exceeds the limit ({})",
-        weight, max_weight
+        display = "Data Request Transaction ({:?}) weight ({}) exceeds the limit ({})",
+        dr_output, weight, max_weight
     )]
-    DataRequestWeightLimitExceeded { weight: u32, max_weight: u32 },
+    DataRequestWeightLimitExceeded {
+        weight: u32,
+        max_weight: u32,
+        dr_output: DataRequestOutput,
```

Comment:
order :rofl:

Reply:
order of appearing in the error message or alphabetical order?